app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable, :trackable
   include DeviseTokenAuth::Concerns::User
+  has_many :rents, dependent: :destry
```

Comment:
`destroy`

Reply:
Then the Book model should also have has_many rents. Do you agree?

Reply:
yes.